test/e2e/package.json

```diff
@@ -34,8 +34,6 @@
     "jasmine-core": "2.99.1",
     "jasmine-reporters": "^2.3.2",
     "regenerator-runtime": "^0.13.7",
-    "wait-on": "^3.3.0",
-    "wdio-iedriver-service": "^0.1.0",
-    "webdriver-manager": "^13.0.0"
```

Comment:
`webdriver-manager` and `wdio-iedriver-service` are not used. 
Removed to git rid of outdated dependencies with vulns (`request`). 
("@wdio/sauce-service" is used for e2e tests in IE instead of IEDriver)